### PR TITLE
aio(fix): api list column overflow fix

### LINKED ISSUE #17250: aio: api list columns overlap
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
API list items lay over each other.

image

**Expected behavior**
API items should not lay on top of each other and should be legible.

**Minimal reproduction of the problem with instructions**
https://aio-staging.firebaseapp.com/api

### PR DESCRIPTION
**Pull request by sjtrimble**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)
Closes https://github.com/angular/angular/issues/17250

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by sjtrimble** [comment]
@IgorMinar @petebacondarwin Not sure if this is acceptable or if would be better to have just one column instead? Let me know what your thoughts are.

File: aio/src/styles/2-modules/_api-list.scss [hunk 1]
```diff
@@ -252,6 +252,7 @@ $form-select-width: 200px;
     padding: 0;
     float: left;
     width: 33%;
+    overflow: hidden;
```
Comment (petebacondarwin) [inline aio/src/styles/2-modules/_api-list.scss]: How about : `text-overflow: ellipsis;`?

Comment (sjtrimble) [inline aio/src/styles/2-modules/_api-list.scss]: It already has that property value on there.

Comment (petebacondarwin) [inline aio/src/styles/2-modules/_api-list.scss]: Oh yeah! Silly me.

**Comment by petebacondarwin** [comment]
So just a bit of margin or padding between the columns would help:

screen shot 2017-06-07 at 18 48 24

**Comment by sjtrimble** [comment]
@petebacondarwin Agreed :)

**Comment by IgorMinar** [comment]
@sjtrimble can you update the commit message to be "fix(aio):" rather than "aio(fix):" thanks.

**Comment by IgorMinar** [comment]
@sjtrimble nvm, I'll just have @alxhub do it during merge. but in the future, please use the correct message format.

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
thanks!

**Pull request merged**